server/src/Main.hs

```diff
@@ -14,15 +14,14 @@ import Control.Monad.Trans.Maybe (MaybeT (MaybeT, runMaybeT))
 import Crypto.Hash (hash)
 import qualified Crypto.WebAuthn.Cose.Registry as Cose
 import qualified Crypto.WebAuthn.Metadata.Service.Types as Service
-import qualified Crypto.WebAuthn.Model as M
 import qualified Crypto.WebAuthn.Model.Binary.Decoding as MD
-import qualified Crypto.WebAuthn.Model.JavaScript as JS
-import Crypto.WebAuthn.Model.JavaScript.Decoding (decodeCreatedPublicKeyCredential, decodeRequestedPublicKeyCredential)
-import Crypto.WebAuthn.Model.JavaScript.Encoding (encodePublicKeyCredentialCreationOptions, encodePublicKeyCredentialRequestOptions)
-import Crypto.WebAuthn.Operations.Assertion (verifyAssertionResponse)
-import qualified Crypto.WebAuthn.Operations.Assertion as M
```

Comment:
@ErinvanderVeen A note for the future: I always recommend manually choosing the modules to import identifiers from, because otherwise you end up with weird things like this without realizing!